maintainers/maintainer-list.nix

```diff
@@ -9660,6 +9660,10 @@
     email = "oliver.huntuk@gmail.com";
     name = "Oliver Hunt";
   };
+  vq = {
+    email = "vq@erq.se";
+    name = "Daniel Nilsson";
```

Comment:
this isn't actually a correct entry to maintainers-list, because it misses the github handle and id, but :ok: